### PR TITLE
Builders package skeleton

### PR DESCRIPTION
**Pull request by alan-agius4**
Note: there is no testing setup yet. But this should be enough to get started

//cc @manfredsteyer

### CODE REVIEW

File: modules/bazel-tsconfig-build.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
     "sourceMap": true,
     "inlineSources": true,
     "target": "es2015",
-    "lib": ["es2015", "dom"],
+    "lib": ["es2018", "dom"],
```
Comment (vikerman) [inline modules/bazel-tsconfig-build.json]: Why was this change needed?

Comment (alan-agius4) [reply modules/bazel-tsconfig-build.json]: Not needed now, removing
